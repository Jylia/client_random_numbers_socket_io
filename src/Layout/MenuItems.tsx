import React, { Component } from 'react'
import {
    Menu,
} from 'semantic-ui-react';

class MenuItems extends Component {

    render() {
        return (
            <React.Fragment>
                <Menu.Item as='a' active>Home</Menu.Item>
                <Menu.Item as='a'>Line Chart</Menu.Item>
                <Menu.Item as='a'>Bar Chart</Menu.Item>
                <Menu.Item as='a'>Threshold play</Menu.Item>
            </React.Fragment>
        )
    }
}

export default MenuItems;