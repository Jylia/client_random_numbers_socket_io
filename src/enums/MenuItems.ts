export enum MenuItemsEnum {
    Home = 1,
    LineChart = 2,
    BarChart = 3,
    TresholdPlay = 4,
};