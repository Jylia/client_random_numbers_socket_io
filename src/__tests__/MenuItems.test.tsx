import React from 'react';
import ReactDOM from 'react-dom';
import MenuItems from '../Layout/MenuItems';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<MenuItems />, div);
  ReactDOM.unmountComponentAtNode(div);
});
