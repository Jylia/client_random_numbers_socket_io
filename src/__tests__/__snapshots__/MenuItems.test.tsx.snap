// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <a
    className="active item"
    onClick={[Function]}
  >
    Home
  </a>,
  <a
    className="item"
    onClick={[Function]}
  >
    Line Chart
  </a>,
  <a
    className="item"
    onClick={[Function]}
  >
    Bar Chart
  </a>,
  <a
    className="item"
    onClick={[Function]}
  >
    Threshold play
  </a>,
]
`;
